engines/dfc_provider/app/controllers/dfc_provider/api/catalog_items_controller.rb

```diff
@@ -1,10 +1,15 @@
 # frozen_string_literal: true
 
 # Controller used to provide the API products for the DFC application
+# CatalogItems are items that are being sold by the entreprise.
 module DfcProvider
   module Api
     class CatalogItemsController < DfcProvider::Api::BaseController
       def index
+        # CatalogItem is nested into an entreprise which is also nested into
+        # an user on the DFC specifications, as defined here:
+        # https://datafoodconsortium.gitbook.io/dfc-standard-documentation
+        #  /technical-specification/api-examples
         render json: current_user, serializer: DfcProvider::PersonSerializer
```

Comment:
I think this would make more sense if it just returned the catalog items, rather than the user.

Reply:
Yes, we still have some issues with granularity on DFC side, but we need to follow the DFC format, not really choosing what to return here. It is the idea to make it simpler, with less data encapsulation, in the next data format versions.

Reply:
Got it, thanks! I'll mark this resolved.